merlin/systems/dag/ops/feast.py

```diff
@@ -285,3 +285,10 @@ def _prefixed_name(cls, output_prefix, col_name):
             return f"{output_prefix}_{col_name}"
         else:
             return col_name
+
+    @property
+    def supported_formats(self) -> DataFormats:
+        return (
+            DataFormats.NUMPY_TENSOR_TABLE
+            | DataFormats.CUPY_TENSOR_TABLE
+        )
```

Comment:
Is this not the default for ops in Systems? I see that some ops are flagged as only supporting tensor tables and some as only supporting data frames. Is the default support for all of those?

Reply:
No we never set it as the default in the inference operator https://github.com/NVIDIA-Merlin/systems/blob/main/merlin/systems/dag/ops/operator.py